### PR TITLE
Fix: updated links to https

### PR DESCRIPTION
**Pull request by stu01509**
Updated links to https, but some link still is broken 404 not found.

http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical
http://terminology.hl7.org/CodeSystem/data-absent-reason
http://hl7.org/fhir/sid/icd-10-cm

### CODE REVIEW

**Comment by stephenwaite** [comment]
hi @stu01509 , looks like some of the urls have changes like
`https://hl7.org/fhir/allergyintolerance.html`
so you can search for them starting here as a base
`https://hl7.org/fhir/`

**Comment by stu01509** [comment]
Hi @stephenwaite 

Thanks for your review and information, I updated commit, Please take a look :)

File: src/Services/FHIR/FhirAllergyIntoleranceService.php [hunk 1]
```diff
@@ -106,7 +106,7 @@ public function getOne($id)
     public function createAllergyIntoleranceResource($id = '', $data = '', $encode = true)
     {
         $typeCoding = new FHIRCoding();
-        $typeCoding->setSystem("http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical");
+        $typeCoding->setSystem("https://hl7.org/fhir/allergyintolerance.html");
```
Comment (yashrajbothra) [inline src/Services/FHIR/FhirAllergyIntoleranceService.php:109]: Hey @stu01509 Nice that you pointed this but u need to know that these URLs are not meant to be visited they are canonical URL which is used for unique identification. We just copy these URLs from fhir Docs for the related ValueSets.
For further lookup, I am adding some links to spec.
https://www.hl7.org/fhir/valueset-allergyintolerance-clinical.html
https://www.hl7.org/fhir/codesystem.html
https://www.hl7.org/fhir/valueset.html

Also, you can ask @sjpadgett for their review 👍 
Thanks 😃

Comment (bradymiller) [reply src/Services/FHIR/FhirAllergyIntoleranceService.php:109]: hi @yashrajbothra , makes sense and gonna close this PR.

**Comment by bradymiller** [comment]
moment of silence:

tenor (11)

**Pull request closed without merging**